.github/workflows/ci.yml

```
@@ -63,3 +73,30 @@ jobs:
           keep_files: true
           external_repository: backpack/backpack.github.io
           publish_branch: master
+
+      - name: Prepare to deploy pull request build
```

I know this was my suggestion, but maybe even more consistent if we do this?
```suggestion
      - name: Prepare to deploy (pull request build)
```

---

It sounds like a song title now 😂

---

CD1:
1. Prepare to deploy
2. Eye of the CI
3. The long road to payload
4. This token has expired

---

5. You're breaking (my build)
6. Prepare to deploy (DJ TypeScript mix)
7. LGTM (feat. Timbaland)